recipes/static/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set package = "static" %}
+{% set version = "1.1.1" %}
+{% set md5 = "75c738ed25cc69a71299b93369ee4e40" %}
+
+package:
+    name: {{ package }}
+    version: {{ version }}
+
+source:
+    fn: {{ package }}-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/source/s/{{ package }}/{{ package }}-{{ version }}.tar.gz
+    md5: {{ md5 }}
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record record.txt
+    preserve_egg_dir: True
+    entry_points:
+        - static=static:command
+
+requirements:
+    build:
+        - python
+        - setuptools
+    run:
+        - python
+        - setuptools
+
+test:
+    imports:
+        - static
+    commands:
+        - static --help
+
+about:
+    home: https://github.com/lukearno/static
+    license: GNU Library or Lesser General Public License (LGPL)
```

Comment:
[Package license]( https://github.com/lukearno/static/blob/v1.1.1/setup.py#L29 ) says MIT. Could we update this to match.